scss/inc/rtl/_action-bar.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2021 Open Assessment Technologies SA
+*/
+
+.runner {
+  .test-runner-scope {
+    .action-bar.content-action-bar.horizontal-action-bar.bottom-action-bar {
```

Comment:
**nitpick:** Is it needed to have such a specificity? It may prevent or make it harder to apply custom theme.